Subject: TOMOYO: Reenable TOMOYO Linux.

Core part has synchronized with TOMOYO 1.8. So, bump version and reenable.
Subsequent patches will be applicable without asking userspace changes.

Signed-off-by: Tetsuo Handa <penguin-kernel@I-love.SAKURA.ne.jp>
---
 security/tomoyo/common.c |    2 +-
 security/tomoyo/tomoyo.c |    3 ---
 2 files changed, 1 insertion(+), 4 deletions(-)

--- security-testing-2.6.orig/security/tomoyo/common.c
+++ security-testing-2.6/security/tomoyo/common.c
@@ -1844,7 +1844,7 @@ static int tomoyo_write_answer(struct to
 static void tomoyo_read_version(struct tomoyo_io_buffer *head)
 {
 	if (!head->r.eof) {
-		tomoyo_io_printf(head, "2.3.0");
+		tomoyo_io_printf(head, "2.4.0");
 		head->r.eof = true;
 	}
 }
--- security-testing-2.6.orig/security/tomoyo/tomoyo.c
+++ security-testing-2.6/security/tomoyo/tomoyo.c
@@ -284,9 +284,6 @@ static int __init tomoyo_init(void)
 
 	if (!security_module_enable(&tomoyo_security_ops))
 		return 0;
-	printk(KERN_INFO "TOMOYO Linux is temporarily disabled due to large "
-	       "and imcompatible changes are in progress.\n");
-	return 0;
 	/* register ourselves with the security framework */
 	if (register_security(&tomoyo_security_ops) ||
 	    init_srcu_struct(&tomoyo_ss))
